docs/appendices/backend-writers-guide.rst

```diff
@@ -729,22 +729,22 @@ other update/remove functionality at a later stage.
 
 .. cpp:function:: virtual bool DNSBackend::startTransaction(const string &qname, int id)
 
-  See :cpp:func:`above <DNSBackend::beginTransaction>`. Please
+  See :cpp:func:`above <DomainInfo::startTransaction>`. Please
   note that this function now receives a negative number (-1), which
   indicates that the current zone data should NOT be deleted.
 
 .. cpp:function:: virtual bool DNSBackend::commitTransaction()
 
-  See :cpp:func:`above <DNSBackend::commitTransaction>`.
+  See :cpp:func:`above <DomainInfo::commitTransaction>`.
```

Comment:
`::commitTransaction()`, `::abortTransaction()` and `::feedRecord()` belong to the `DNSBackend` class.

Reply:
(same as other comment)